src/logic/compile.ts

```diff
@@ -33,7 +33,7 @@ export async function compile(
 ): Promise<void> {
   for (const profile of profiles) {
     //Compile profile
-    if (!options?.onlyMap) {
+    if (!options?.onlyMap && profile.path !== undefined) {
```

Comment:
suggestion: add test to check profile compilation is skipped for remote profile. It can be done in follow up PR.